webapi/src/main/scala/org/knora/webapi/responders/Responder.scala

```diff
@@ -148,8 +148,8 @@ abstract class Responder(responderData: ResponderData) extends LazyLogging {
      * @param iriFormatter the stringFormatter method that must be used to create a random Iri.
      * @return IRI of the entity.
      */
-    protected def checkEntityIri(entityIri: Option[SmartIri],
-                                 iriFormatter: => IRI): Future[IRI] = {
+    protected def checkOrCreateEntityIri(entityIri: Option[SmartIri],
```

Comment:
@subotic I renamed the function. As you can see it is a 9 line long function and it is actually very clear what it does. I do not see how I can make it even shorter, since checking the custom IRI and creation of random Iri are already happening in other functions. This function checks if a custom Iri is given; if yes, calls the `checkIriExists` function. If no, calls the `makeUnusedIri` function passing the `stringformatter` method that should be used for this purpose.

Reply:
The name is much clearer now. Thank you.